content/learning-paths/servers-and-cloud-computing/processwatch/understanding-processwatch.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Understanding Process Watch
+weight: 4
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Understanding Process Watch
+At a high level the Process Watch tool works like so
```

Comment:
This is phrased in a quite informal way, could you rephrase it more like technical documentation?